### ISSUE TITLE
Influence between Reader and app?

### ISSUE THREAD

**Issue by MasterPlexus** [vicktor/FreeStyleLibre-NFC-Reader#4]
Hi, just to clarify : 
- Did using the app influence the Reader in some cases?
- I read here in Wiki that the Tag are only readable with the reader which activated the tag... Did the App works anyway?
- Are there any progress since January?
Best regards, and many thanks for trying this coding!
SP

**Comment by funtax** [comment]
I got my FSL a few days ago and I'm also an Android-developer, interested in working on this.
So.. I just have the same questions and am looking forward to the answers :)

**Comment by LightSnowDev** [comment]
>Did using the app influence the Reader in some cases?

Yes it does in some cases. But i don't think all of them are listed.

>I read here in Wiki that the Tag are only readable with the reader which activated the tag... Did the App works anyway?

The app works anyway. It is not diffucult to get the information. You can read the data with any NFC application. Only in a very few cases it failes.

>Are there any progress since January? Best regards, and many thanks for trying this coding! SP

I don't know. The main problem is the alogorithm. Look here: http://type1tennis.blogspot.com.es/search/label/Freestyle%20Libre
